Class 'FirebaseFirestoreService' incorrectly implements class 'Firestore' [error TS2720:]

I have an issue when trying to run nest build:

```
node_modules/@aginix/nestjs-firebase-admin/dist/services/firebase-admin-firestore.service.d.ts:2:22 - error TS2720: 
Class 'FirebaseFirestoreService' incorrectly implements class 'Firestore'. Did you mean to extend 'Firestore' and 
inherit its members as a subclass?
  Property 'recursiveDelete' is missing in type 'FirebaseFirestoreService' but required in type 'Firestore'.

2 export declare class FirebaseFirestoreService implements FirebaseFirestore.Firestore {
                       ~~~~~~~~~~~~~~~~~~~~~~~~

  node_modules/@google-cloud/firestore/types/firestore.d.ts:278:5
    278     recursiveDelete(
            ~~~~~~~~~~~~~~~
    'recursiveDelete' is declared here.

Found 1 error(s).
```

And i have this config for nestjs-firebase-admin , nest core and nest CLI:

```
"@aginix/nestjs-firebase-admin": "^2.1.0",
"@nestjs/common": "^7.5.1",
"@nestjs/core": "^7.5.1",
"@nestjs/cli": "^7.5.1",
```
And I tested this on node v12.17.0, v12.18.1, and on the latest but with no success. Thank you for your feedback.

--- a/lib/services/firebase-admin-firestore.service.ts
+++ b/lib/services/firebase-admin-firestore.service.ts
@@ -60,4 +60,11 @@ export class FirebaseFirestoreService implements FirebaseFirestore.Firestore {
   bundle(bundleId?: string): FirebaseFirestore.BundleBuilder {
     return this.firestore.bundle(bundleId);
   }
+
+  recursiveDelete(
+    ref: FirebaseFirestore.CollectionReference<unknown> | FirebaseFirestore.DocumentReference<unknown>,
+    bulkWriter?: FirebaseFirestore.BulkWriter,
+  ): Promise<void> {
+    return this.firestore.recursiveDelete(ref, bulkWriter);
+  }
 }

--- a/package.json
+++ b/package.json
@@ -28,7 +28,8 @@
     "prepublish:npm": "npm run build",
     "publish:npm": "npm publish --access public",
     "prepublish:next": "npm run build",
-    "publish:next": "npm publish --access public --tag next"
+    "publish:next": "npm publish --access public --tag next",
+    "prepare": "npm run build"
   },
   "peerDependencies": {
     "@nestjs/common": "^7.1.2",

--- a/package.json
+++ b/package.json
@@ -38,7 +38,7 @@
     "firebase-admin": "^9.5.0"
   },
   "devDependencies": {
-    "@nestjs/common": "^7.4.2",
+    "@nestjs/common": "^7.6.15",
     "@nestjs/core": "^7.4.2",
     "@nestjs/testing": "^7.4.2",
     "@types/jest": "^26.0.10",
@@ -51,7 +51,7 @@
     "reflect-metadata": "^0.1.13",
     "rimraf": "^3.0.0",
     "supertest": "4.0.2",
-    "ts-jest": "24.3.0",
+    "ts-jest": "^26.5.6",
     "tslint": "^6.1.3",
     "typescript": "3.9.7"
   },

